scripts/lib/setup-apt-repo

```diff
@@ -44,13 +44,16 @@ SOURCES_FILE=/etc/apt/sources.list.d/$LIST.list
 PREF_FILE=/etc/apt/preferences.d/$LIST.pref
 STAMP_FILE=/etc/apt/sources.list.d/$LIST.list.apt-update-in-progress
 
-ZULIP_SCRIPTS="$(cd "$(dirname "$(dirname "$0")")" && pwd)"
-LIST_PATH="$ZULIP_SCRIPTS/setup/apt-repos/$LIST"
+# All paths, for purposes of hashing, should be relative to the deploy
+# root.
+cd "$(dirname "$(dirname "$(dirname "$0")")")"
```

Comment:
We should never chain more than one `dirname` call—I know we do in places, but we shouldn’t, because it breaks in case of the directories is `.`.

```suggestion
cd "$(dirname "$0")/../..
```

Reply:
Wow that was fast. ☺

Reply:
I was already re-looking at what I'd written with my head tilted. :)